app/services/idv/agent.rb

```diff
@@ -27,7 +27,7 @@ def proof_resolution(
         user_id: user_id,
         threatmetrix_session_id: threatmetrix_session_id,
         request_ip: request_ip,
-        double_address_verification: double_address_verification,
+        ipp_enrollment_in_progress: ipp_enrollment_in_progress,
```

Comment:
[my previous comment was wrong](https://github.com/18F/identity-idp/pull/9390/files#r1364528036), the key here needs to keep the old name to ensure a smooth deploy
```suggestion
        double_address_verification: ipp_enrollment_in_progress,
```